docs/data/base/components/select/select.md

```diff
@@ -0,0 +1,105 @@
+---
+product: base
+title: React Select unstyled component and hook
+components: SelectUnstyled, MultiSelectUnstyled, OptionUnstyled, OptionGroupUnstyled
+githubLabel: 'component: select'
+waiAria: https://www.w3.org/TR/wai-aria-practices/#combobox
+packageName: '@mui/base'
+---
+
+# Select
+
+<p class="description">The SelectUnstyled and MultiSelectUnstyled components let you create menus of options for users to choose from.</p>
```

Comment:
```suggestion
<p class="description">The select components allow developers to create menus of options for users to choose from.</p>
```

We are already mentioning below the name of the components, so it's redundant. Also, to keep consistent with other docs pages, I have rephrased a bit.

Reply:
According to https://github.com/mui/material-ui/pull/31850#discussion_r830077088, "allow" is not the best word to use here. How about "The select components let you create..."

Also, I'm not sure about the word "menus" here. I understand it may be used in a broader sense than "a menu component", but I'm afraid it could confuse developers and make them wonder which component should be used to actually implement a menu control.